lib/src/great_circle.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:math' as math;
+import 'package:turf/turf.dart';
+import 'helpers.dart';
+
+/// Calculates the great circle route between two points on a sphere 
+/// 
+/// Useful link: https://en.wikipedia.org/wiki/Great-circle_distance
+ 
+List<List<double>> greatCircle(
+  dynamic start,
```

Comment:
instead of dynamic please choose a more concrete type (the JS version asks for coordinates, which would equal to our `Position` type